src/jit/compiler.h

```diff
@@ -2298,6 +2298,8 @@ public :
     
     unsigned            lvaLocAllocSPvar;               // variable which has the result of the last alloca/localloc
 
+    unsigned            lvaNewObjArrayArgs;             // variable with arguments for new MD array helper
```

Comment:
I think it would be useful to add a comment here explaining this in a bit more detail - i.e. why and how does it represent multiple arguments in a single variable?